config/lib/helpers/broadcast.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 module.exports = {
-  broadcastContentTypes: [
-    'broadcast',
-  ],
+  contentTypes: {
+    askChangeTopicBroadcast: {
```

Comment:
I like how the content types are setup in the broadcast helper config. I think switching `topics` and `defaultTopicTriggers` to this scheme would simplify them (by adding more consistency) a bit more. Maybe is too much changes for this PR, but something to consider in future PRs.